controller.go

```diff
@@ -221,6 +222,8 @@ func loadSettings() error {
 		Default(aws.DefaultHealthCheckInterval.String()).DurationVar(&healthCheckInterval)
 	kingpin.Flag("health-check-timeout", "sets the health check timeout for the created target groups. The flag accepts a value acceptable to time.ParseDuration").
 		Default(aws.DefaultHealthCheckTimeout.String()).DurationVar(&healthCheckTimeout)
+	kingpin.Flag("healthy-threshold-count", "The number of consecutive health checks successes required before considering an unhealthy target healthy").
+		Default(strconv.FormatUint(aws.DefaultHealthyThresholdCount, 10)).UintVar(&healthyThresholdCount)
```

Comment:
We need to think more about defaults:
* it is 5 for ALB https://docs.aws.amazon.com/elasticloadbalancing/latest/application/target-group-health-checks.html
* it is 3 for NLB https://docs.aws.amazon.com/elasticloadbalancing/latest/network/target-group-health-checks.html

One way is to have two flags `alb-healthy-threshold-count` and `nlb-healthy-threshold-count` (like we have `alb-http-target-port` and `nlb-http-target-port`).

Reply:
Nice, good catch. I recall encountering that when doing some other testing, too.

Reply:
There is also `UnhealthyThresholdCount` which is default 2 for ALB but must be equal to `HealthyThresholdCount` for NLB (not sure if it breaks when not specified). I guess we can leave it out of scope of this change until needed.

Reply:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-targetgroup.html#cfn-elasticloadbalancingv2-targetgroup-healthythresholdcount

Reply:
Thanks @AlexanderYastrebov for that research, I missed that. I can add that prefix. Should we leave the NLB part out for now, as not required?

Reply:
@AlexanderYastrebov scope is now reduced to ALBs only, the pattern is set so it can be extended easily if needed. Or do you like to do the full setup of all conditions now?

Reply:
> the pattern is set so it can be extended easily if needed

True, I do not think the full set of flags is necessary atm. 
@szuecs WDYT?

Reply:
We can go with ALB only, but on the other hand I would recommend using NLBs anyways, so why not include it.
If you only need limited scope feel free to reduce the scope to ALB.

Reply:
Not a deal, I've added the support for ALBs and NLBs including their conditions. 👍🏻 
Hope that's it, let me know if there is anything else you like to have adjusted

Reply:
:+1: Thank you